invenio_oauthclient/contrib/cern_openid.py

```diff
@@ -363,7 +362,6 @@ def account_setup(remote, token, resp):
     resource = get_resource(remote, resp)
 
     with db.session.begin_nested():
-        person_id = resource.get("cern_person_id")
```

Comment:
unused

Reply:
this is interesting, we have person_id in ILS, not sure now how it gets there

Reply:
🤷‍♂️ I just see that the `person_id` is extracted and never used hehe